lib/sort_by_length.rb

```diff
@@ -1,7 +1,28 @@
 # A method which will return an array of the words in the string
 #  sorted by the length of the word.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n^2)
+# Space complexity: O(n)
 def sort_by_length(my_sentence)
```

Comment:
This works, but because of the way you're doing it with `insert`, this will be O(n^3) time complexity.  I strongly encourage you to look at my examples of the sorts specifically how I swapped individual elements without needing to use insert or so many external arrays.